db/models/assessment_record/assessment_records.py

```diff
@@ -49,6 +50,10 @@ class AssessmentRecord(db.Model):
         "workflow_status", ENUM(Status), index=True, default="NOT_STARTED"
     )
 
+    asset_type = db.Column(
+        "asset_type", db.String(255), index=True, nullable=False
```

Comment:
Why is this a String here but modelled as an enum?

Reply:
@gidsg Sorry, this was a draft PR just to help us look at the changes. Ive just removed the enum stuff now